ledger/participant-integration-api/src/main/resources/db/migration/h2database-appendonly/V1__Append_only_schema.sql

```diff
@@ -121,9 +121,16 @@ CREATE TABLE participant_command_completions (
     application_id VARCHAR NOT NULL,
     submitters ARRAY NOT NULL,
     command_id VARCHAR NOT NULL,
+    -- The transaction ID is `NULL` for rejected transactions.
     transaction_id VARCHAR,
-    status_code INTEGER,
-    status_message VARCHAR
+    -- The rejection status is `NULL` if the completion is for an accepted transaction.
+    -- The `rejection_status` contains a Protocol-Buffers-serialized message of type
```

Comment:
```suggestion
    -- The rejection status contains a Protocol-Buffers-serialized message of type
```

(Or citing the column name everywhere for uniformity)